mig-agent/config_test.go

```diff
@@ -0,0 +1,182 @@
+package main
+
+import (
+	"fmt"
+	"testing"
+
+	"gopkg.in/gcfg.v1"
+)
+
+func TestConfigLoadDefault(t *testing.T) {
+	// loading the default config produces an error
+	expect := `configLoad() -> config.Certs.Ca open /path/to/ca/cert: no such file or directory`
+	err := configLoad("../conf/mig-agent.cfg.inc")
```

Comment:
I mean more if you do something like `go test mig.ninja/mig/mig-agent` from some other directory, not using the makefile